lib/constants.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @enum {number}
+ */
+const DEFAULT_LEVELS = {
+  trace: 10,
+  debug: 20,
+  info: 30,
+  warn: 40,
+  error: 50,
+  fatal: 60
+}
+
+/**
+ * @enum {"ASC" | "DESC"}
```

Comment:
I am unfamiliar with this syntax. Can you point me to the documentation in https://jsdoc.app where it describes using string values for the type like this?

Reply:
I'm sorry, there is no exact mention of that, but [https://jsdoc.app/tags-type](here) exists a note: _"Full support for Google Closure Compiler-style type expressions is available in JSDoc 3.2 and later."_

Reply:
The linked docs also do not show this sort of declaration.

Reply:
Changed